src/AppBundle/QueryType/ChildrenQueryType.php

```diff
@@ -35,6 +35,11 @@ public function getQuery(array $parameters = [])
                 $parameters['sorting_order'] == 'desc' ? Query::SORT_DESC : Query::SORT_ASC
             );
         }
+        else if (isset($parameters['sorting_clause']) &&
```

Comment:
Can we do something more generic for the `sortClauses` with the `QueryType` and make it compatible with all the cases?